src/lib/webworker.generated.d.ts

```diff
@@ -226,9 +230,9 @@ declare var Event: {
 }
 
 interface EventTarget {
-    addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+    addEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean): void;

```

Comment:
TIL `listener` is optional. Although not sure what's the point in allowing this :/